.storybook/src/data.ts

```diff
@@ -1,4 +1,15 @@
-import { poslog, cementData, holeSizeData, casingData, surfaces, completion, stratColumns, picks, seismic } from './esv-intersection-data';
+import {
+  poslog,
+  cementData,
+  holeSizeData,
+  casingData,
+  surfaces,
+  completion,
+  stratColumns,
+  picks,
+  seismic,
+  completionV2,
```

Comment:
Let delete existing completion data in esv-intersection-data repo and place our new data there. So we call it "completion" not "completionV2"